Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM mhart/alpine-node:8.0.0
+
+MAINTAINER PRX <sysadmin@prx.org>
+LABEL org.prx.app="yes"
+
+# install git, aws-cli
+RUN apk --no-cache add git ca-certificates \
+    python py-pip py-setuptools groff less && \
+    pip --no-cache-dir install awscli
+
+# install PRX aws-secrets scripts
+RUN git clone -o github https://github.com/PRX/aws-secrets
+RUN cp ./aws-secrets/bin/* /usr/local/bin
+
+ENV TINI_VERSION v0.9.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static /tini
+RUN chmod +x /tini
+
+ENV APP_HOME /app
+ENV PHANTOM true
+RUN mkdir -p $APP_HOME
+WORKDIR $APP_HOME
+EXPOSE 4202
+
+ENTRYPOINT [ "/tini", "--", "./bin/application" ]
+CMD [ "serve" ]
+
+ADD ./package.json ./
+ADD ./yarn.lock ./
+RUN apk --update add curl git && \
+  curl -Ls "https://github.com/dustinblackman/phantomized/releases/download/2.1.1/dockerized-phantomjs.tar.gz" | tar xz -C / && \
+  npm install yarn --global && \
+  yarn install --no-progress --silent && \
+  apk del curl && \
```

Comment:
I see publish has a `npm cache clean && \` after this line, is that not a good idea? should it be removed from publish too?

Reply:
It's not a thing anymore. It wasn't causing problems in Publish yesterday, but it should be removed.
```
$ npm cache clean
npm ERR! As of npm@5, the npm cache self-heals from corruption issues and data extracted from the cache is guaranteed to be valid. If you want to make sure everything is consistent, use 'npm cache verify' instead.
npm ERR! 
npm ERR! If you're sure you want to delete the entire cache, rerun this command with --force.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/sandibarr/.npm/_logs/2017-10-04T15_01_28_056Z-debug.log
```

Reply:
I had no idea, thanks!